### ISSUE TITLE
AdWords - FOI Made Easy & Make an FOI Request campaign maintenance

### ISSUE THREAD

**Issue by henare** [openaustralia/righttoknow#679]
Extracted from #660:

>* Right to Know - FOI Made Easy + Right to Know – Make an FOI Request
>    * These campaigns could be combined into 1 and the keywords should be reorganised into tighter Ad Group themes
>    * Ad Copy should then be tailored specifically to each Ad Group

From our session today:

* Extract low performing useful sets of keywords into new Ad Groups, e.g. "human rights", with new, targeted Ads
* Remove low performing keywords from existing FOI made easy Ad Group

**Comment by henare** [comment]
>These campaigns could be combined into 1 and the keywords should be reorganised into tighter Ad Group themes

I've just been battling trying to do this in AdWords Editor. When copying one Ad Group to another it would say the ads were "Invalid. Edit to post.". After a lot of fiddling I've realised it's because they're standard text ads, which you [can no longer create or edit](https://support.google.com/adwords/answer/7048854?hl=en-GB&authuser=1).

So another important thing to do is to update all these ads to expanded text ads. They perform better and are more likely to be served by Google so that's quick win too :trophy:

**Comment by henare** [comment]
>These campaigns could be combined into 1 and the keywords should be reorganised into tighter Ad Group themes

To combine these two campaigns I've:

* Removed duplicate keywords so that there's not overlaps between the ad groups when they become part of the same campaign - there were duplicates due to capitalisation in _Make an FOI request_ and 3 that overlapped - I left those only in _FOI made easy_ as it's doing better right now
* Copied the _Make an FOI request_ Ad Group into the _FOI made easy_ campaign
* Renamed the _FOI made easy_ campaign to just _FOI_ since it's more generic now
* Paused the unused _Make an FOI request_ campaign
* In the _Make an FOI request_ Ad Group ads:
  * Deleted the 1 paused ad since we didn't want to import an unused ad
  * Created expanded text ads for the 2 active ads that were as similar as possible to the previous text ads

**Comment by henare** [comment]
>Extract low performing useful sets of keywords into new Ad Groups, e.g. "human rights", with new, targeted Ads

Since we've just merged _Make an FOI request_ and changed its ads to expanded text ads I've decided to leave that Ad Group alone and let it settle in.

On _FOI made easy_ I paused 2 keywords about departments that were not displaying due to a low quality score. So far I've found keywords around "FOI letters/forms" and "human rights" are good ones to split out and create new Ad Groups for with targeted ads so that's the next step.

**Comment by henare** [comment]
I've made a new Ad Group called, _FOI request form_. It has 16 keywords I moved from _FOI made easy_ and 7 from _Make an FOI request_. I've created 4 ads that contain the main different keywords, "Request form", "Request letter", "Sample request", and "Request template".


I've also made an Ad Group called, _Human/civil rights_ and moved 44 keywords there. It has 3 ads broadly on the topics of, "human rights", "civil rights", and "freedom and rights".

**Comment by henare** [comment]
I've disabled the 2 remaining standard text ads. One was being displayed to less than 1% of visitors so I didn't replace that and the other one I replaced with a similar expanded text ad.

**Comment by henare** [comment]
I've added sitelinks - "Find An Authority", "Make A Request", "Browse Existing Requests", "Get Help Making Requests".

I think that's everything done so I'll close this now.

**Comment by henare** [comment]
It's been a week so I checked in on how these changes are doing.

4,310 impressions generated 242 clicks and there's been 5 conversions this week (all up on the week before that). 2 of those conversions were requests (same as the week before). CTR is good, around the same at 5.61%.

_Make an FOI Request_ is the worst performing group with 28 clicks and 0 conversions. About a 3rd of its keywords are low search volume so that may explain it. I'm surprised that _Human/civil rights_ is doing so well - 1 request made and the most clicks out of all ad groups in this campaign (109 clicks). The 2 ad groups I edited seem to be performing better now so :+1:
